recipes/default.rb

```diff
@@ -15,3 +15,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+include_recipe 'yum-epel'
+
+package 'scponly'
+
```

Comment:
In order for the chroot command to work, the binary needs to be setuid. You can do that by doing the following:
```ruby
file '/usr/sbin/scponlyc' do
  mode '4755'
end
```